### PR TITLE
ICMSLST-1203 - SIL - Response Preparation - incorrect Cover Letter templates offered.

### PR DESCRIPTION
**Pull request by chris-pettinga**
Before:
image

After:
image

### CODE REVIEW

File: web/management/commands/utils/add_template_data.py [hunk 1]
```diff
@@ -1237,48 +1237,20 @@ def add_letter_templates():
...
     )
     Template.objects.get_or_create(
-        start_datetime=pytz.timezone("UTC").localize(
-            dt.datetime.strptime("05-JAN-2016 09:56:09", DATETIME_FORMAT), is_dst=None
-        ),
         is_active=True,
-        template_name="Imported into Northern Ireland (Section 5)",
+        template_name="Firearms SIIL Cover Letter",
```
Comment (marcuspp) [inline web/management/commands/utils/add_template_data.py:1249]: Typo `SIL`

File: web/management/commands/utils/add_template_data.py [hunk 2]
```diff
@@ -1237,48 +1237,20 @@ def add_letter_templates():
...
...n the authority given to you under Article 45 of the Firearms    (Northern Ireland) Order 2004, limits the number of Article 6(1) weapons, ammunition and component parts which you    may hold at any one time. It is an offence to be in breach of this authority.</p>  <p align="justify">Please note that the change to import licensing requirements from 1 January is that imports of    firearms and deactivated firearms into GB from the EU are treated the same as imports from non-EU countries into the    UK.</p>  <p align="justify">The import licensing requirements for transfers from the EU to Northern Ireland will remain    unchanged under the terms of the Northern Ireland Protocol. This means that<u>,</u> subject to specified    exclusions<u>,</u> shipments from the EU to Northern Ireland will continue to require a transfer licence issued by    the transferring EU Member State.</p>  <p align="justify">There is also no change to the import licensing requirements for imports from non-EU countries into    the UK.</p>  <p align="justify">Notices to Importers detailing the current import licensing controls are published on <u>www.gov.uk/guidance/import-controls#notices-to-importers</u></p>',
```
Comment (marcuspp) [inline web/management/commands/utils/add_template_data.py:1253]: Dynamic variables not set correctly

Comment (MattHolmes123) [reply web/management/commands/utils/add_template_data.py:1253]: More examples here: `licence issued by    the `

Comment (chris-pettinga) [reply web/management/commands/utils/add_template_data.py:1253]: @marcuspp can you please explain a little more?

**Review by MattHolmes123 — APPROVED** [review]

**Review by marcuspp — APPROVED** [review]

**Pull request merged**